Cargo.toml

```diff
@@ -31,6 +31,7 @@ members = [
     "metrics",
     "net-shaper",
     "notifier",
+    "program-interface",
```

Comment:
I think `program-interface` won't live in here.  It would be a stand-alone crate with it's own `[workspace]`

Reply:
Yeah, that is temporary, everything outside of `/program-interface` is temp